help/configure-unread-counts-for-streams.md

```diff
@@ -0,0 +1,29 @@
+# Configure unread counts for streams
+
+Zulip lets you configure how and when to display stream unread counts.
+You can choose whether unread counts are displayed for both muted and unmuted
+streams in the left sidebar. If you find them distracting, you can configure
+this setting so that a dot is shown, instead of an unread count, to indicate
+when there is an unread message in the stream, and you can reveal the unread
+count by hovering over the stream.
+
```

Comment:
How about something like this?

---

# Configure unread message counters for streams

To provide information about unread messages, Zulip can show unread message counters next to stream names in the left sidebar. This is likely to be helpful if you generally read (or otherwise [mark as read](/help/marking-messages-as-read)) all the messages in a stream, but may otherwise feel unnecessary.

To provide flexibility, Zulip offers an option to display a dot instead of an unread counter for streams in the left sidebar. You can use a dot for all streams, or just for streams you have [muted](/help/mute-a-stream). You will still be able to see the number of unread messages in a stream by moving your mouse over it in the left sidebar.